src/omen/cogs/wvw_commands.py

```diff
@@ -162,6 +174,69 @@ async def get_skirmish_data(self, match_data):
+        remaining_skirmishes = forecaster.predict(steps=(84 - data['skirmish_id'][-1]))
+
+        # create a complete match representation based on the forecast
+        match = pandas.concat([past_skirmishes, remaining_skirmishes])
+
+        # for debugging and understanding, log the match
+        pandas.set_option('display.max_rows', None)
+        self.logger.debug(f"forecast after {data['skirmish_id'][-1]} skirmishes:")
+        self.logger.debug(f"\n{match}")
+
+        # forecast a winner with victory points
+        # TODO This is horrible, but I wanted to get it working fast
+        for skirmish in match.index:
+            if match['green_score'][skirmish] > match['blue_score'][skirmish] and match['blue_score'][skirmish] > match['red_score'][skirmish]:
+                self.match_data_dict['green']['predicted_victory_points'] += 5
+                self.match_data_dict['blue']['predicted_victory_points'] += 4
+                self.match_data_dict['red']['predicted_victory_points'] += 3
+            elif (match['green_score'][skirmish] > match['red_score'][skirmish] and match['red_score'][skirmish] > match['blue_score'][skirmish]):
+                self.match_data_dict['green']['predicted_victory_points'] += 5
+                self.match_data_dict['red']['predicted_victory_points'] += 4
+                self.match_data_dict['blue']['predicted_victory_points'] += 3
+            elif (match['blue_score'][skirmish] > match['red_score'][skirmish] and match['red_score'][skirmish] > match['green_score'][skirmish]):
+                self.match_data_dict['blue']['predicted_victory_points'] += 5
+                self.match_data_dict['red']['predicted_victory_points'] += 4
+                self.match_data_dict['green']['predicted_victory_points'] += 3
+            elif (match['blue_score'][skirmish] > match['green_score'][skirmish] and match['green_score'][skirmish] > match['red_score'][skirmish]):
+                self.match_data_dict['blue']['predicted_victory_points'] += 5
+                self.match_data_dict['green']['predicted_victory_points'] += 4
+                self.match_data_dict['red']['predicted_victory_points'] += 3
+            elif (match['red_score'][skirmish] > match['blue_score'][skirmish] and match['blue_score'][skirmish] > match['green_score'][skirmish]):
+                self.match_data_dict['red']['predicted_victory_points'] += 5
+                self.match_data_dict['blue']['predicted_victory_points'] += 3
+                self.match_data_dict['green']['predicted_victory_points'] += 3
+            elif (match['red_score'][skirmish] > match['green_score'][skirmish] and match['green_score'][skirmish] > match['blue_score'][skirmish]):
+                self.match_data_dict['red']['predicted_victory_points'] += 5
+                self.match_data_dict['green']['predicted_victory_points'] += 4
+                self.match_data_dict['blue']['predicted_victory_points'] += 3
+            else:
```

Comment:
There's probably a better way of doing this in Python. I'll figure it out, but I wanted a look at the concept and the ability to get predictions locally, at least, and get some feedback on the overall idea.